pom.xml

```
@@ -1,57 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-
-  <parent>
-    <groupId>org.jenkins-ci.plugins</groupId>
-    <artifactId>plugin</artifactId>
-    <version>1.424</version>
-  </parent>
-
-  <artifactId>copy-to-slave</artifactId>
-  <packaging>hpi</packaging>
-  <name>Copy To Slave Plugin</name>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/Copy+To+Slave+Plugin</url>
-  <version>1.4.5-SNAPSHOT</version>
-
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-  </properties>
-
-  <developers>
-    <developer>
-      <id>svvivek</id>
-      <name>Vivekanand S V</name>
-      <email>svvivekanand@gmail.com</email>
-      <timezone>+5.5</timezone>
-    </developer>
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>3.4</version>
+        <relativePath />
+    </parent>
+    <groupId>io.jenkins.plugins</groupId>
+    <artifactId>copy-to-slave</artifactId>
+    <version>1.4.8</version>
+    <packaging>hpi</packaging>
+    <properties>
+        <jenkins.version>1.625</jenkins.version>
```

```suggestion
        <jenkins.version>2.60.3</jenkins.version>
```

If you want to support Java 8 only, we need to use the core version which supports Java 8 only

---

Do we need Java 8 here?